core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml

```diff
@@ -1,9 +1,9 @@
 ---
-- name: Wait for kubectl to find and access a Kubernetes cluster
+- name: Wait for kubectl to find and access K8s cluster
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl cluster-info
-  retries: 50
+  retries: 30 # 1min
```

Comment:
Won't be those comments misleading if someone will forget to change number of minutes after changing retries?